stable/datadog/templates/daemonset.yaml

```diff
@@ -20,6 +21,7 @@ spec:
 {{ toYaml .Values.resources | indent 12 }}
         ports:
         - containerPort: 8125
+          hostPort: 8125
```

Comment:
I agree this is useful, but most deployments don't need it and enabling it by default might cause port conflicts on some setups.
I'd prefer to have it linked to a configuration option and disabled by default.